### PR TITLE
osu! performance points balancing pass

### PR DESCRIPTION
**Pull request by stanriders**
Changes included here:

1. Slightly raw aim buff (which does **not** include slider aim or high bpm aim)
2. Add OD multipliers to ok/meh miss additions for relax (https://www.desmos.com/calculator/bc9eybdthb), remove ar bonus and reduce flashlight pp for relax (replaces #16708)
3. Reduce speed 50s nerf (would love to remove it completely eventually, but for now just https://www.desmos.com/calculator/axwv6i9hrs)
4. Reduce low AR bonus because some maps were getting more pp with easy mode than without it (i.e https://osu.ppy.sh/beatmapsets/149471 +HTEZ being 391pp vs +HT being 379pp)

---

SR/PP changes sheet: https://docs.google.com/spreadsheets/d/1M26JmN76yLo9H8MxDrlNz0Aeh2p4BhaT6Mnt4gux7IU/edit

As of https://github.com/ppy/osu/pull/19001/commits/760742e358a3a12d995bdf544baf87871a178df3

### CODE REVIEW

**Review by emu1337 — DISMISSED** [review]
Global aim multiplier doesn't seem necessary but doesn't break anything either.

File: osu.Game.Rulesets.Osu/Difficulty/OsuPerformanceCalculator.cs
Comment (apollo-dw) [inline osu.Game.Rulesets.Osu/Difficulty/OsuPerformanceCalculator.cs:59]: OD can actually be higher than 12 with 2.0x @ OD11, [which means we can get negative `effectiveMissCount`.](https://i.imgur.com/gCLIENr.png) Max OD should be used here, though I'm not sure what that is off the top of my head.

Comment (stanriders) [reply osu.Game.Rulesets.Osu/Difficulty/OsuPerformanceCalculator.cs]: image
Fixed, OD 13.33 was chosen because its an OD at which point great hitwindow becomes 0

**Comment by smoogipoo** [comment]
SR/PP sheet added to description. Awaiting 2 approvals before merge.

**Review by apollo-dw — APPROVED** [review]

**Review by emu1337 — DISMISSED** [review]

**Review by apollo-dw — CHANGES_REQUESTED** [review]
Going to unapprove in light of player satisfaction, I think we should consider looking into increasing the PP multiplier and this is a good PR to do it on

**Review by apollo-dw — APPROVED** [review]

**Comment by apollo-dw** [comment]
[As per frost's comment in pp-dev,](https://discord.com/channels/546120878908506119/619282780563570688/999103905889800393) the cube root term in the total SR formula needs to be updated to reflect any changes to the global pp multiplier. Making the multiplier static and using it instead of updating it everytime (and potentially forgetting to) sounds like the right course of action to me.

https://github.com/ppy/osu/blob/633f6fe62082bfdea5e5be38434e233b6a9d6777/osu.Game.Rulesets.Osu/Difficulty/OsuDifficultyCalculator.cs#L67

**Review by apollo-dw — CHANGES_REQUESTED** [review]
As above

**Review by emu1337 — DISMISSED** [review]

**Review by apollo-dw — DISMISSED** [review]

**Review by smoogipoo — APPROVED** [review]

**Pull request merged**
